components/ResourceInformation.js

```diff
@@ -0,0 +1,68 @@
+import { useOperation } from "react-openapi-client";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import Chip from "@mui/material/Chip";
+import Rating from "@mui/material/Rating";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import { Box, CircularProgress } from "@mui/material";
+import UserRating from "./UserRating";
+
+function ResourceInformation({ resourceId }) {
+  console.log(resourceId);
+  const { loading, data, error } = useOperation("getResource", resourceId);
+
+  if (loading) {
+    return (
+      <Box sx={{ display: "flex" }}>
+        <CircularProgress />
+      </Box>
+    );
+  }
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  }
+
+  const average = () => {
+    const ave = data.average_evaluation;
+    if (!ave) {
+      return null;
+    }
+
+    return parseFloat(ave.slice(0, 3));
+  };
```

Comment:
Is it really necessary to create a method for this?
Why would the average be null?
Also, the variable names are very confusing, what is average and what is ave?

Reply:
You're right, is not necessary a method for this. I refactored it and created as a variable with a one-liner if-else. I changed the names a little bit, but keep it short. Also, moved this file to the folder `components/resources`

The API return a null average value if the resource has not been evaluated yet. That could be changed, but is out of scope for this PR.